services/workflows-service/src/providers/file/file-provider/aws-s3-file.service.ts

```diff
@@ -167,7 +167,7 @@ export class AwsS3FileService implements IStreamableFileProvider {
     return `https://${bucketName}.s3.amazonaws.com/${fileName}`;
   }
 
-  generateRemoteFilePath(fileName: string, directory?: string): string {
-    return `${directory !== undefined ? `${directory}/` : ''}${fileName}`;
+  generateRemoteFilePath(fileName: string, customerName: string, directory?: string): string {
```

Comment:
Using one object parameter would make this function usages more readable
```suggestion
  generateRemoteFilePath({fileName, customerName, directory}: {fileName: string, customerName: string, directory?: string}): string {
```